### DISCUSSION TITLE
快速暂停使用MouseInc

### DISCUSSION THREAD

**Discussion by raccoon666666** [shuax/MouseInc.Settings#234] [Ideas]
MouseInc有个坑的地方就是,  即使排除了某exe,  但边缘角触发和全局热键却没有排除, 
所以打游戏时总是出现鼠标移动到右下显示到了桌面的问题(我是这么设置边缘角的)
所以每次打游戏就得手动暂停MouseInc,  太瘠薄麻烦了
能不能通过命令行暂停MouseInc
这样我就可以设置全局热键开快速开关MouseInc了

**Comment by shuax** [comment]
不是有现成的Ctrl+Alt+M

**Reply by Comestones** [reply]
thx!
